### PR TITLE
Omit `_serialization.py` from code coverage

### PR DESCRIPTION
**Pull request by mccoyp**
# Description

`_serialization.py` is a common, generated file that has a good amount of unused code in most applications.

This PR removes the `--omit` flag from the `coverage xml` command that's used to generate our code coverage reports, and instead uses only the `.coveragerc` file at the root of the repository. Using the command flag overrides options in the configuration file, which is confusing and led to prior efforts to omit `_serialization.py`.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by mccoyp** [comment]
Example of a code coverage report without `_serialization.py` omission: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2713304&view=codecoverage-tab

Example of one _with_ `_serialization.py` omission: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2713509&view=codecoverage-tab

File: scripts/devops_tasks/create_coverage.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def collect_tox_coverage_files():
 def generate_coverage_xml():
     if os.path.exists(coverage_dir):
         logging.info("Generating coverage XML")
-        commands = ["coverage", "xml", "-i", "--omit", '"*test*,*example*"']
+        commands = ["coverage", "xml", "-i"]
```
Comment (scbedd) [inline scripts/devops_tasks/create_coverage.py:56]: Pls double check that it applies the exclusion correctly (automatically finding the coveragerc I mean).

Looks g2g other than that question!

**Review by scbedd — APPROVED** [review]
One single comment but g2g!

File: scripts/devops_tasks/create_coverage.py (hunk 1 above)
Comment (mccoyp) [reply scripts/devops_tasks/create_coverage.py:56]: Verified with some pipeline runs here: https://github.com/Azure/azure-sdk-for-python/pull/29972#issuecomment-1515526565 🙂

**Pull request merged**